Snippets/Rabbit/Rabbit_4/Usage.cs

```diff
@@ -92,7 +92,7 @@ void UseRoutingTopology(EndpointConfiguration endpointConfiguration)
         #region rabbitmq-config-useroutingtopology
 
         var transport = endpointConfiguration.UseTransport<RabbitMQTransport>();
-        transport.UseRoutingTopology<MyRoutingTopology>();
+        transport.UseRoutingTopology(durableExchangesAndQueues => new MyRoutingTopology());
```

Comment:
@Particular/rabbitmq-transport-maintainers this is the only non-trivial change. Do you think the expression parameter name is OK?

Also, the snippet will no longer work for 4.0, only 4.1. Does that matter?